stories/Board.stories.tsx

```diff
@@ -1,7 +1,7 @@
 import React, { useState } from 'react';
 import { StoryFn, Meta } from '@storybook/react';
 import Board from '../src/Boards/Board';
-import { ChartProps, Theme } from '../src/interfaces/propInterfaces';
+import { ChartProps, Theme } from '../src';
```

Comment:
i don't think we should be referencing index.ts internally, I'd rather be explicit with internal code on where something is being imported from.  To me index.ts is just for exporting to the consumer of this component set.